packages/features/ee/organizations/pages/settings/appearance.tsx

```diff
@@ -147,6 +153,32 @@ const OrgAppearanceView = ({
         }}
         switchContainerClassName="mt-6"
       />
+
+      <SettingsToggle
+        toggleSwitchAtTheEnd={true}
+        title={t("seo_indexing")}
+        description={t("allow_seo_indexing")}
+        disabled={mutation.isPending}
+        checked={allowSEOIndexingValue}
+        onCheckedChange={(checked) => {
+          setAllowSEOIndexingValue(checked);
+          mutation.mutate({ allowSEOIndexing: checked });
+        }}
+        switchContainerClassName="mt-6"
+      />
+
+      <SettingsToggle
+        toggleSwitchAtTheEnd={true}
+        title={t("disable_org_url_label")}
+        description={t("disable_org_url_description")}
```

Comment:
AFAIK organizations cannot see what their autoAcceptEmail are. Can you pass the current org url and orgAutoAcceptEmail so that they exactly where they are being redirected to.

So description text will be dynamically generated different for different orgs like
`Redirects acme.cal.com to acme.com` for acme,  and then `Redirects dunder-mifflin.cal.com to dunder-mifflin.ai` etc.

Reply:
I meant whatever `${orgSlug}.cal.com` and not current page url.

Reply:
even if we say try to hard code to show current org url dynamically from autoAcceptEmail - getting `acme.com` and inserting `cal` in the middle, this may not be 100% accurate i feel, in all cases.

Reply:
> I meant whatever `${orgSlug}.cal.com` and not current page url.

yeah ok, this must be possible will try to implement and update.

Reply:
@Amit91848 , updated to show description dynamically and also tested once.